### PR TITLE
Contact Form 

### PR DESCRIPTION
**Pull request by Therealadetunji**
Modified html and css codes to match the sample on figma.

### CODE REVIEW

File: Portfolio.html [hunk 1]
```diff
@@ -1,225 +1,325 @@
...
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <div class="tag-box">
+              <ul class="tags">
+                <li class="tag">HTML</li>
+                <li class="tag">CSS</li>
+                <li class="tag">Javascript</li>
+              </ul>
+            </div>
```
Comment (AmaduKamara) [inline Portfolio.html:111]: - [ ] You've done well so far. Kindly check the broken images in your design and sort them out so your portfolio continues to look great as it is in other parts so far. Please check the image paths and fix them so all images load properly.

**See the broken images in your app**
image

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️♻️♻️ 

Hi @Therealadetunji,

AlonzoLeroneThisCloseGIF (3)

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✅ Correct setup of Gitflow
✅ No linter error
✅ Contact form works fine

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark: :heavy_check_mark:

Hi @Therealadetunji  ,  

Your project is complete! Great job for making all the requested changes 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- Validations correctly implemented
- No linter errors
- images loading
- form submits successfully

<br>
Cheers.Happy coding!..and keep soaring higher! 💻 🍷 🚀 

great

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Meltrust — COMMENTED** [review]
Hi @Therealadetunji,

## Wow, you did it 🎉 
Brilliant

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### To highlight

- The form works✔️ 
- Flexbox was used✔️ 
- Made the changes requested by last week's reviewers✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**